data/assistant-templates/example.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "example",
+  "name": "Example Assistant",
+  "description": "This is an *example assistant* for testing purposes.",
+  "sample_questions": [
```

Comment:
Detalj, men kanske ska vara camelCase?

Reply:
mjo, men den läser in rakt av genom pydantic och vill inte ha extra `sampleQuestions` -> `sample_questions` konvertering...jsonen används bara en gång vid setup också.